examples/3Drendering/objloader.py

```diff
@@ -132,7 +132,7 @@ def __init__(self, filename, swapyz=False):
 def MTL(filename):
     contents = {}
     mtl = None
-    return
```

Comment:
Actually, did you test this with a mtl file? Because I'm not sure the MTL function is currently working as is.